src/app/services/baw-api/baw-api.service.ts

```diff
@@ -360,34 +375,49 @@ export type InnerFilter<T = {}> = Combinations<T> &
  * https://github.com/QutEcoacoustics/baw-server/wiki/API:-Filtering
  */
 export interface Filters<T = {}, K extends keyof T = keyof T> {
+  /** Filter settings */
   filter?: InnerFilter<T>;
+  /** Include or exclude keys from response */
   projection?: {
+    /** Whitelist keys */
```

Comment:
```suggestion
    /** include keys */
```

Reply:
Is this inclusion? Its a list of all the keys which will be returned, not a list of additional keys to include?

Reply:
it is inclusion.

By default, the filter has an implicit 'include' everything. If you override it, it only returns the explicit list you asked for.